### ISSUE TITLE
Switch to build manylinux2014 wheels

### ISSUE THREAD

**Issue by nsoranzo** [galaxyproject/starforge#230]
manylinux1 reached EOL and manylinux2010 is based on CentOS 6 (EOL). See https://github.com/pypa/manylinux for the differences.

xref. https://github.com/galaxyproject/starforge-recipes/pull/46#issuecomment-494792808

**Comment by natefoo** [comment]
Rather than put any effort into Starforge, I think we probably ought to switch to cibuildwheel. The only issue I am aware of is that cibuildwheel is mainly designed for one project per repo (for projects to build the wheels for their package to upload to PyPI) whereas that will be fairly cumbersome and unwieldy for us. I am not sure how difficult/possible it would be to make a single repo building multiple projects using cibuildwheel.

**Comment by nsoranzo** [comment]
@natefoo I agree, I think we could work on the idea proposed in https://github.com/pypa/cibuildwheel/issues/597 and rework/recreate this repo as cibuildwheel recipes.
